stubs/Markdown/markdown/util.pyi

```diff
@@ -41,15 +41,15 @@ class HtmlStash:
 
 class Registry:
     def __init__(self) -> None: ...
-    def __contains__(self, item): ...
+    def __contains__(self, item: Any): ...
     def __iter__(self) -> Any: ...
-    def __getitem__(self, key): ...
+    def __getitem__(self, key: str): ...
```

Comment:
This is incorrect. It also accepts `str` and `slice`: https://github.com/Python-Markdown/markdown/blob/12c3378ab9a46fae416a7bb5549e3f123a11d22a/markdown/util.py#L313-L322

```suggestion
    @overload
    def __getitem__(self, key: slice) -> Registry: ...
    @overload
    def __getitem__(self, key: str | int) -> Any: ...
```